core/src/main/java/org/apache/accumulo/core/client/mapreduce/InputFormatBase.java

```diff
@@ -325,27 +362,12 @@ protected void setupIterators(TaskAttemptContext context, Scanner scanner, Strin
      *          the Hadoop context for the configured job
      * @param scanner
      *          the scanner to configure
+     * @deprecated since 1.7.0; Use {@link #contextIterators} instead.
      */
     @Deprecated
     protected void setupIterators(TaskAttemptContext context, Scanner scanner) {
-      setupIterators(context, scanner, null);
-    }
-
-    /**
-     * Initialize a scanner over the given input split using this task attempt configuration.
-     */
-    protected void setupIterators(TaskAttemptContext context, Scanner scanner, org.apache.accumulo.core.client.mapreduce.RangeInputSplit split) {
```

Comment:
`setupIterators` with that sig became concrete in super type, `AbstractInputFormat` 
https://github.com/apache/accumulo/pull/25/files#diff-696314c306e81784094e2dc76341728bR479

I understood that would keep the compatability.